modules/_labs/header/react/stories/stories.tsx

```diff
@@ -333,7 +333,7 @@ storiesOf('Labs|Header/React', module)
       <div css={containerStyle}>
         <Header
           variant={Header.Variant.Dub}
-          title="Centered Menu"
+          title="Centered Menu With Search"
```

Comment:
nb Looks like this added text is what's pushing the Logout button over the boundary on the screenshot https://www.chromaticqa.com/snapshot?appId=5d854c26ba934e0020f5e98a&id=5e2b41d359313d0022640f60

Reply:
since the responsive stuff is by prop the smaller screenshots can get squished or overflow

Reply:
IE11 handles this lack of space differently. IE11 is squishing the navigation tabs while Chrome and Firefox are pushing the "Logout" button to the right.

Should the Header or the navigation tabs handle not having enough space? Maybe by compressing into a dropdown menu instead? Or is the app supposed to detect this and do something different?

Reply:
I think user of the component user should decide and pass isCollapsed at the correct time. I think that allows for the most flexibility and makes it so the header doesn't have to do expensive measuring of all different parts anytime there is a change.